evil-core.el

```diff
@@ -874,7 +874,7 @@ one already."
                                   (keymap-parent map)
                                   state)
```

Comment:
You may as well remove this `state` as well?

Reply:
Good catch! I removed it in the new commit.